Jenkinsfile

```diff
@@ -475,8 +490,23 @@ pipeline {
             description: "")
         booleanParam(
             name: "PERF_TEST",
-            defaultValue: false,
+            defaultValue: true,
             description: "Enable performance testing stages")
+        booleanParam(
+            name: "PERF_TEST_FP16",
+            defaultValue: true,
+            description: "Enable performance testing stages")
+        booleanParam(
+            name: "PERF_TEST_FP32",
+            defaultValue: true,
+            description: "Enable performance testing stages")
```

Comment:
@alexandraBara @JehandadKhan After this change, `PERF_TEST` and `PERF_TEST_FP16/FP32` are not orthogonal anymore. I recommend:
- Either re-using existing `DATATYPE_FP16/FP32` instead of `PERF_TEST_FP16/FP32` (and getting rid of these vars)
- Or, if the functionality provided by the above is insufficient, get rid of `PERF_TEST`, so we'll simply have `PERF_TEST_FP16` and `PERF_TEST_FP32`.